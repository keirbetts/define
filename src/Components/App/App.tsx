import "./App.css";
import { Word } from "./Word/Word";


const App = () => {

  return (
    <div className="App">
      <Word />
    </div>
  );
};

export default App;
