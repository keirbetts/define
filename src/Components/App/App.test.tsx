import App from "./App";
import { render } from "@testing-library/react";

describe("<App />", () => {
  it("renders successfully", async () => {
    const { container } = render(<App />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
