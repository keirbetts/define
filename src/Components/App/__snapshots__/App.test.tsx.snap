// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders successfully 1`] = `
<div
  class="App"
>
  <div
    class="word-wrapper"
  >
    Define
    <br />
    <br />
    <br />
    <br />
    <textarea
      placeholder="Enter word definition..."
    />
    <br />
    <br />
    <button>
      Submit guess
    </button>
  </div>
</div>
`;
